app/store/itinerary-search-store.coffee

```
@@ -176,7 +176,12 @@ class ItinerarySearchStore extends Store
 
   storeItinerarySearch: (data) ->
     @data = data
-    window?.sessionStorage?.setItem STORAGE_KEY, JSON.stringify @data
+    try
+      window?.sessionStorage?.setItem STORAGE_KEY, JSON.stringify @data
+    catch error
+      if error.name == 'QuotaExceededError'
```

else re-throw, to not hide unexpected errors?